### ISSUE TITLE
PPSSPP

### ISSUE THREAD

**Issue by tonyunreal** [ThatGuySam/doesitarm#347]
**The official name of the app**
PPSSPP

**Is there a supported build available on a public/release channel?**
No

**Proposed App Status**
Yes, works via Rosetta 2 with native support in development

**Proposed App Category**
Entertainment and Media Apps

**Related Issue Tracker Link or discussion**
Issue: https://github.com/hrydgard/ppsspp/issues/13708

**An Official App Download Page**
https://build.ppsspp.org/?page/downloads#osx

**Full Screenshot with the App and Activity Monitor(if self reporting) or [Silicon Info](https://github.com/billycastelli/Silicon-Info)**
![Screen Shot 2020-12-02 at 3 53 15 PM](https://user-images.githubusercontent.com/3460639/100844341-b75dc480-34b6-11eb-8d69-ca240f8e6c8e.png)
![Screen Shot 2020-12-02 at 3 50 13 PM](https://user-images.githubusercontent.com/3460639/100844358-bcbb0f00-34b6-11eb-8b48-db80d0d6f56a.png)

**Comment by ThatGuySam** [comment]
Thanks for the App Request! 

it looks like they confirmed that it doesn't work natively but they didn't say if it works via Rosetta 2. 

Looks like we'll need a screenshot or another tracker link to verify. 

Be sure to include the App, Activity Monitor showing the app details, and About this Mac so we can see the exact OS version. 

You can find an Activity Monitor example on #176

**Comment by tonyunreal** [comment]
Updated OP with proper status and screenshots.

For sdl2 I [installed the x64 version of Homebrew](https://soffes.blog/homebrew-on-apple-silicon), ran "arch -x86_64 brew install sdl2 libzip", then PPSSPP worked fine.

**Comment by ThatGuySam** [comment]
App is now live on the site! 

https://doesitarm.com/app/ppsspp/

Feel free to browse through the other [M1 Issues](https://github.com/ThatGuySam/doesitarm/issues?q=is%3Aopen+is%3Aissue+label%3A%22Needs+M1+Testing%22) to help with missing info on other apps.

**Comment by tonyunreal** [comment]
PPSSPP 1.11 was released a few days ago, while there is no mention of Apple Silicon in the patch notes, I found out that [Homebrew has a formula of PPSSPP](https://formulae.brew.sh/formula/ppsspp) and the formula info page shows that it has a arm64 bottle ready for install. 

So I installed it with Homebrew (along with about 30 dependency packages, yike), and surprisingly it worked fine, no crash or slowdown or anything. The only missing part was Vulkan support, which turn out to be due to an outdated dynamic-linked MoltenVK library that the PPSSPP source code (and in turn this pre-built binary) ships with, I replaced it with a newer, universal build of MoltenVK then Vulkan worked fine.

I have asked in the official bug tracker whether this can be recognized as official Apple Silicon support, if the answer is affirmative I will submit an update here.
